packages/dx-react-chart/docs/guides/legend_title.md

```diff
@@ -0,0 +1,27 @@
+# React Chart - Legend and Title
+
+The legend and title provide additional information about a chart.
+
+## Legend
+
+The Legend feature displays series on the current chart. Legend contains items. Items contain marker and label. Marker display series color. Label text equal series name.
+
+## Title 
+
+Title display helpful text. You must provide text to the title.
+
+## Related Plugins
```

Comment:
Let's move this section up to make it the first one like it is done in other guides.
Do we really need separate sections for _Legend_ and _Title_? There is no much to say about them and it would be also strange to create separate demos for them. So, I believe we can just extend the common description a little bit and do not create these sections at all.